k9mail-library/src/main/java/com/fsck/k9/mail/store/imap/ImapUtility.java

```diff
@@ -241,9 +239,13 @@ static Long extractHighestModSeq(ImapResponse imapResponse) {
     static List<String> extractVanishedUids(List<ImapResponse> imapResponses) {
         List<String> uids = new ArrayList<>();
         for (ImapResponse imapResponse : imapResponses) {
-            if (imapResponse.getTag() == null && ImapResponseParser.equalsIgnoreCase(imapResponse.get(0), "VANISHED")
-                    && imapResponse.isString(1)) {
-                uids = ImapUtility.getImapSequenceValues(imapResponse.getString(1));
+            if (imapResponse.getTag() == null && ImapResponseParser.equalsIgnoreCase(imapResponse.get(0), "VANISHED")) {
```

Comment:
can you add examples of the imap response lines that are expected and parsed here as a comment? I think that gives the reader a better impression of what's going on than deducting the format from the lines :)